docs/sync.js

```diff
@@ -468,6 +468,58 @@ class Session {
     removeProgressNotification(progressCallback) {}
 }
 
+/**
+ * An object encapsulating partial sync subscriptions.
+ * @memberof Realm.Sync
+ */
+class Subscription {
+    /**
+     * Gets the current state of the subscription.
+     * Can be either:
+     *  - Realm.Sync.SubscriptionState.Error: An error occurred while creating or processing the partial sync subscription.
+     *  - Realm.Sync.SubscriptionState.Creating: The subscription is being created.
+     *  - Realm.Sync.SubscriptionState.Pending: The subscription was created, but has not yet been processed by the sync server.
+     *  - Realm.Sync.SubscriptionState.Complete: The subscription has been processed by the sync server and data is being synced to the device.
+     *  - Realm.Sync.SubscriptionState.Invalidated: The subscription has been removed.
+     * @type {number}
+     */
+    get state() {}
+
+    /**
+     * Gets the error message. `undefined` if no error.
+     * @type {string}
+     */
+    get error() {}
+
+    /**
+     * Unsubscribe a partial synced `Realm.Results`. The state will change to `Realm.Sync.SubscriptionState.Invalidated`.
+     * The `Realm.Results` will not produce any meaningful values.
```

Comment:
It'd be worth mentioning here that any objects matching this query will be removed if they are not matched by any other query, and that they'll be removed asynchronously with respect to this method.